src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -64,6 +71,8 @@ class PanelMediaBrowser extends LitElement {
 
   @state() _currentItem?: MediaPlayerItem;
 
+  @state() _preferredLayout: MediaPlayerLayoutType = "auto";
```

Comment:
Can we store this in local storage?

Reply:
Certainly could, though not sure if that's the best for the user.

I was kind of thinking that auto is the right answer for 99% of the time, but only in some exceptional cases (like where you can't read a long filename) you might want to switch.

But then do you really want it to be switched to list by default if you come back later and try to select some different media? Or is it simplest to just always start from auto mode. Since this will affect all views in the media browser.

Reply:
Yeah, I just typed a story about storing it per media id, but removed it when I was finished writing it...

Reply:
Let's keep it default to auto for now.